gcovr/writer/html.py

```diff
@@ -367,34 +384,47 @@ def print_html_report(covdata: CovData, output_file, options):
         data["css_link"] = css_link
 
     data["theme"] = options.html_theme
+    data["row_sort_key"], data["row_sort_reverse"] = row_sort_keys(options)
```

Comment:
There's a lot in this PR that would be part of the other PR. I'll do my best to differentiate those changes to a new PR.

Reply:
In my opinion it's not needed to move the sorting to the Jinja level and change the kind of sorting. Removing this would remove a little bit of complexity.

Changeing the sorting to a multilevel sorting in general can be done in a separate PR.